libs/sdk-ui-tests-e2e/recordings/mappings/gdc_app_account_bootstrap-c1b99964-52a5-47cf-986e-72f05462afdf.json

```diff
@@ -0,0 +1,22 @@
1hcyr8i8lo8/obj/13258\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13261\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13264\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13267\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13270\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13274\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13277\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13287\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13615\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13620\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13630\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13688\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13702\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13706\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13709\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13749\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13765\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13772\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13776\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13826\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13859\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13863\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13908\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13914\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13937\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13959\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13982\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/13990\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14005\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14055\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14060\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14066\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14084\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14141\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14153\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14161\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14165\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14168\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14191\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14338\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14365\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14372\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14386\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14390\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14395\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14407\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14417\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14424\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14433\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14463\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14480\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14539\",\"/gdc/md/mbuumy476p78ybcceiru61hcyr8i8lo8/obj/14547\"],\"mapboxToken\":\"pk.eyJ1IjoiZ29vZGRhdGEiLCJhIjoiY2s4YTN4NW5zMDBxdjNncGN5cTZ5dTBpbCJ9.36wcS9BKyaN1NzGBrwLw1w\"}}}",
```

Comment:
You might want to delete your phone number from here.

Reply:
Is there anything like Rail user for staging3? It would be much better if we have these recordings made under this user instead of personal accounts.

Reply:
There is not afaik :-/